vespa-feed-client/pom.xml

```diff
@@ -46,6 +46,11 @@
       <artifactId>junit-jupiter</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.github.tomakehurst</groupId>
+      <artifactId>wiremock-jre8-standalone</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
I think this one was not intended either.

Reply:
It was, but I forgot to add the test.